ref/Castle.Core-net45.cs

```diff
@@ -2615,6 +2615,7 @@ public class InvalidProxyConstructorArgumentsException : System.ArgumentExceptio
     {
```

Comment:
Any idea why `SerializableAttribute` doesn't show up on these classes? PublicApiGenerator defect?

Reply:
Looks like it. PublicApiGenerator is using Mono to reflect over types, and since `SerializableAttribute` is a pseudo-attribute (it gets turned into the `TypeAttributes.Serializable` bit IIRC) it won't show up like the others. I'll likely submit a PR to them.

Shall we go ahead with this (no new regression here) or wait for a fixed version of PublicApiGenerator (might take some time)?

Reply:
Ah yer, forgot about it being a metadata flag.

Reply:
PublicApiGenerator 10.1.1 is up on Nuget

Reply:
Cheers @danielmarbach, thanks for the quick turnaround.